Tamil/Thirukural.md

```diff
@@ -1,6 +1,18 @@
 ﻿முகநக நட்பது நட்பன்று நெஞ்சத்து அகநக நட்பது நட்பு
 Face's smile doesn't shows the friendship, but the heart's rejoice does.
 
-உடுக்கை இழந்தவன் கைபோல ஆங்கே 

```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.